src/controllers.cpp

```diff
@@ -73,7 +75,22 @@ std::unique_ptr<Controller> getControllerById(const unsigned short& id)
     }
     case GAME_CONTROLLER_ID:
     {
-        controller.reset(new GameController());
+        try
+        {
+            /* try to load the game controller with the next level of the
+               playing serie manager, if the file cannot be opened, this
+               constructor throws a std::invalid_argument exception */
+            controller.reset(new GameController());
+
+            /* catch the invalid argument exception if the game cannot start; we
+               get it as a reference to make the program runs faster */
+        }
+        catch(std::invalid_argument&)
+        {
+
+            /* render the error controller instead of the game controller */
+            controller.reset(new ErrorController());
```

Comment:
check if it can be refactored, we have the exact same instruction somewhere else